src/git/install.sh

```diff
@@ -61,40 +93,66 @@ receive_gpg_keys() {
     fi
 }
 
-apt_get_update()
-{
-    if [ "$(find /var/lib/apt/lists/* | wc -l)" = "0" ]; then
-        echo "Running apt-get update..."
-        apt-get update -y
+pkg_mgr_update() {
+    if type apt >/dev/null 2>&1; then
+        if [ "$(find /var/lib/apt/lists/* | wc -l)" = "0" ]; then
+            echo "Running apt-get update..."
+            apt-get update -y
+        fi
+    elif [ ${INSTALL_CMD} = "dnf" ] || [ ${INSTALL_CMD} = "yum" ]; then
+        if [ "$(find /var/cache/${INSTALL_CMD}/* | wc -l)" = "0" ]; then
+            echo "Running ${INSTALL_CMD} check-update ..."
+            ${INSTALL_CMD} check-update
+        fi
     fi
 }
 
+
 # Checks if packages are installed and installs them if not
 check_packages() {
-    if ! dpkg -s "$@" > /dev/null 2>&1; then
-        apt_get_update
-        apt-get -y install --no-install-recommends "$@"
+    if type apt >/dev/null 2>&1; then
```

Comment:
Should we use `INSTALL_CMD` in all cases for consistency?

Reply:
Added commit to use `INSTALL_CMD` in all cases.